test/probot.test.ts

```diff
@@ -571,5 +571,21 @@ describe("Probot", () => {
         expect(error.message).toMatch(/error from app/);
       }
     });
+
+    it.only("passes logger to webhooks", async () => {
```

Comment:
Should the `.only` be there? Or is it just from your local testing?

Reply:
😅

Reply:
damn how did that get back in here, sorry, I'll do a follow up